src/07-even-more-loops.js

```diff
@@ -12,8 +12,15 @@
  *  shoutForLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutForLoop(array) {}
+function shoutForLoop(array) {
+  
+  for(let element of array) {
+    console.log(`element!`)
 
+   return array;
```

Comment:
Looks like our loops just returns the array on first iteration.  Walk through what exactly is happening on each iteration of your loop to figure out what we need to do to solve.